travis/setup_dependencies_common.sh

```diff
@@ -50,7 +50,7 @@ conda create $QUIET -n test python=$PYTHON_VERSION
 source activate test
 
 # EGG_INFO
-if [[ $SETUP_CMD == egg_info ]]; then
+if [ $SETUP_CMD == 'egg_info' ] && [ -z $FORCE_DEPENDENCIES_INSTALL ]; then
```

Comment:
I would prefer it to have a bool value like the rest of the similar variables (e.g. DEBUG), and test in this line for the value of it.

Also this needs a short description at the bottom of the travis section in the readme.